src/TKEBudgetTerms.jl

```diff
@@ -285,6 +285,44 @@ function KineticEnergyDiffusiveTerm(model; location = (Center, Center, Center))
 end
 #---
 
+#--- Kinetic energy forcing term
+@inline function uᵢFᵤᵢᶜᶜᶜ(i, j, k, grid, forcings,
+                                         clock,
+                                         model_fields)
+
+    uFᵘ = ℑxᶜᵃᵃ(i, j, k, grid, ψf, model_fields.u, forcings.u, clock, model_fields)
+    vFᵛ = ℑyᵃᶜᵃ(i, j, k, grid, ψf, model_fields.v, forcings.v, clock, model_fields)
+    wFʷ = ℑzᵃᵃᶜ(i, j, k, grid, ψf, model_fields.w, forcings.w, clock, model_fields)
+
+    return uFᵘ+ vFᵛ + wFʷ
+end
+
+"""
+    $(SIGNATURES)
+
+Return a `KernelFunctionOperation` that computes the forcing term of the KE prognostic equation:
+
+```
+    FORC = uᵢFᵤᵢ
+```
+
+where `uᵢ` are the velocity components and `Fᵤᵢ` is the forcing term(s) in the `uᵢ`
+prognostic equation (i.e. the forcing for `uᵢ`).
+"""
+function KineticEnergyForcingTerm(model::NonhydrostaticModel; location = (Center, Center, Center))
+    validate_location(location, "KineticEnergyForcingTerm")
+    model_fields = fields(model)
+
+    if model isa HydrostaticFreeSurfaceModel
+        model_fields = (; model_fields..., w=ZeroField())
+    end
```

Comment:
If there isn't a function that generalizes for different models, we should add it in Oceananigans.